components/Title.tsx

```diff
@@ -13,6 +13,10 @@ type Props = {
   style?: StyleProp<ViewStyle>;
 };
 
+function navToEmergencyView(pressed: GestureResponderEvent, mainNav: NavigationProp<TabRoutes>): void {
+  mainNav.navigate('EmergencyNumber') as unknown;
+}
+
```

Comment:
Hat bestimmt einen guten Grund, wollte aber mal fragen wofür du die das GestureResponderEvent brauchst und die navigation damit in einer separaten Funktion machst?